src/components/modeler/Modeler.vue

```diff
@@ -529,6 +529,15 @@ export default {
         return this.paper.findViewByModel(shape);
       });
     },
+    getElementByNodeId(id) {
+      const cells = this.paper.model.getCells();
+      for (const cell of cells) {
+        if (cell.component && cell.component.id === id) {
+          return cell;
+        }
+      }
```

Comment:
this could be refactored to a single `find` (lodash or vanilla javascript).
Returns the object if found, cant remember what returns if the element is not found.
Vanilla Javascript: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find
https://lodash.com/docs/4.17.15#find